frontend/src/app/core/services/event.service.ts

```diff
@@ -14,4 +15,20 @@ export class EventService {
     public userChanged(user: UserDto | undefined) {
         this.onUserChanged.next(user);
     }
+
+    private onChangesLoaded = new Subject<DatabaseItem[] | undefined>();
+
+    public changesLoadedEvent$ = this.onChangesLoaded.asObservable();
+
+    public changesLoaded(item: DatabaseItem[] | undefined) {
+        this.onChangesLoaded.next(item);
+    }
+
+    private onChangesSaved = new Subject<string | undefined>();
+
+    public changesSavedEvent$ = this.onChangesSaved.asObservable();
+
+    public changesSaved(guid: string | undefined) {
+        this.onChangesSaved.next(guid);
+    }
```

Comment:
QC: sort class members: public members go first, fields and properties before methods